Miscellaneous Coalition Missions

Arachi-Lover opened the pull request:
adds a compilation of ~~6~~ 9 short-medium mission chains to Coalition space, each too small for their own PR so I am doing these all at a time.

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+				`	"Sorry, should I have paid first before eating?"`
+			branch translator
+				has "language: Coalition"
+			`	The Kimek doesn't have a translation box, so your words don't reach him, and while he insists in remaining by the table, you also notice he has been constantly glancing at the entrance, as if waiting for someone to come in.`
```
Fairly sure it should be insists on not insists in

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+			`	The Kimek doesn't have a translation box, so your words don't reach him, and while he insists in remaining by the table, you also notice he has been constantly glancing at the entrance, as if waiting for someone to come in.`
+			`	When a Heliarch agent passes by the entrance, the Kimek says something in his language, heading to meet the Heliarch, and speaking with them briefly. The Heliarch agent then accompanies the Kimek waiter to your table.`
+			choice
+				`	"Is there an officer problem?"`
```
Is there a problem officer (Or is it intentional?)

downRightCoder256 replied:
you could suggest a commit, you know. makes it easier on him.

Arachi-Lover replied:
was intentional as a joke

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+		conversation
+			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Rinworlds have no day or night, but looking at the conventioned time used here, it is almost night time when Chikee returns to your ship, in a hurry.`
```
Ringworlds not Rinworlds, Not sure if conventioned is right.

Arachi-Lover replied:
suggestion for a word that would be better than conventioned, given the context?

Rafocale replied:
Maybe change it to be based off your ship's internal clock as your reference?

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Rinworlds have no day or night, but looking at the conventioned time used here, it is almost night time when Chikee returns to your ship, in a hurry.`
+			`	"Accepted I was! Grant me the rannk, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`
```
Rank not rannk

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Rinworlds have no day or night, but looking at the conventioned time used here, it is almost night time when Chikee returns to your ship, in a hurry.`
+			`	"Accepted I was! Grant me the rannk, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`
+			`	You congratulate him on the achievement, and wish him best of luck in his new line of work, as he leaves your ship one last time.`
```
wish him *the* best of luck

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+			label vaccines
+			`	He shows you to a detailed map of Kimek space, explaining that most of the confirmed and suspected cases have been there, as one would expect given that the Kimek population is more than double that of the Arachi and the Saryds combined.`
+			`	"To <destination>, this initial batch must be brought. From there, distributing more batches of the vaccines to other worlds, you will be tasked with."`
+			`	As the line of volunteering merchant captains continues to grow behind you, he is switft to inform you that payment will be <payment>, and asks that the next person come forward.`
```
swift not switft

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+		"coalition jobs" ++
+		conversation
+			`An incessant stream of freighters has the landing pads of the spaceport here overcrowded, and from the looks of it you only got a quick clearance due to carrying the vaccines.`
+			`	The people here who are surprisingly managing to keep the place organized despite the overwhelming traffic pay you <payment>, as they unload the vaccines off your cargo hold.`
```
Sentence flow here feels a bit off, feels like some commas might be needed.

Arachi-Lover replied:
is it better now?

Rafocale replied:
I was thinking more along the lines of "The people here, who are surprisingly managing to keep the place organized despite the overwhelming traffic, pay you <payment> as they unload the vaccines off your cargo hold."

Arachi-Lover replied:
that's... what I did yes. think I can take that as a yes then? :v

Rafocale replied:
Sorry, appears that the version I read wasn't the updated one. Ye, it's good now.

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+			label work
+			`	They tell you that they are students at Starlit University, and will be graduating soon. They intend on throwing a party to celebrate, and had the idea of buying some traditional Arachi Rum for that.`
+			`	"Stopped shipping it here, the Arachi have," another Saryd says, pulling on the translation box on his friend's neck, nearly yanking it off. "Happened with many other drinks, this also has."`
+			`	They explain that the rum is produced on <destination>, and that you'll probably have an easier time finding it there. Though you can't know for sure how many people they intend on inviting to this party, and although you don't know how much of a drink a Saryd can have, you still worry a bit when they say that "two tons should do."`
```
"How much a saryd can drink" might be the better way of phrasing that.

Rafocale commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+			label shops
+			`	You try and entertain yourself by looking at the shops here, but pretty much all of them are just filled with the produce of the farms here.`
+			`	One of them has a few toys made out of straw, you take a look at the Saryd designs for them, though they're nothing particularly special, nor as stereotypical as one might have expected from a species that resembles centaurs. Most of them appear to be replicating creatures of the local fauna, or buildings like small houses.`
+			`	You notice two other census workers have come back, so you decide to return to the group and wait for the final member. Some Coalition captains from early have gathered together in a bar to lament the long wait.`
```
From earlier

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,584 @@ mission "Coalition License Patch"
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`
+			`	"Accepted I was! Grant me the rank, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`
+			`	You congratulate him on the achievement, and wish him the best of luck in his new line of work, as he leaves your ship one last time.`
```
wdym? the last paragraph?

Arrow2thekn33 replied:
To me, it feels like, for a mission chain a few missions long, it should have a little bit more of a resolution. As of now, you congratulate him and say bye for the last time all in one sentence. For the ending of a mission chain, this seems too short to me.

Arachi-Lover replied:
I've added some stuffs to it, would you say it is better now?

Arrow2thekn33 replied:
Yep, that’s definitely better.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,584 @@ mission "Coalition License Patch"
+
+mission "Saryd Census 2"
+	name "Saryd Census"
+	description "Transport these <bunks> Saryds census workers to <destination>, where they will collect data on the population growth there."
```
Saryds is the plural for them, would it be changed still?

Arrow2thekn33 replied:
Yes, because in this case “Saryd” is an adjective describing the workers.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,584 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`A young Saryd bumps into you as you're passing by a nearby shop, knocking you to the ground.`
+			`	They help you up, apologizing. A couple that were with him entered the shop without even noticing you, but almost immediately come back out.`
```
this actually refers to a couple other Saryd students, not a couple as in the relationship term. adding "others" after the "couple" to make it clear, since I forgot apparently

Arachi-Lover commented:
I've just added 3 more sets of missions (one for each of the species, for a total of 3 for each now.)
for the Saryd Couple  missions, Jana asked for an option to tell them they don't need to pay you, and as far as I know the only way I could do that is if I added a choice for it with an `apply` `set` for some condition, and base the final, landing mission variants on that condition, so I'll be doing that later. though if there was an easier way to do that, it'd be nicer, so tell me if there is.

BlueFireball commented on data/coalition/coalition missions.txt:
```diff
@@ -724,3 +897,61 @@ mission "Arachi Orphans"
+			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`
+			choice
+				`	"Of course! It's called <ship>."`
+				`	"The name is nothing special, I wouldn't even get the reference it if I read your comic in the future honestly."`
```
seems like you forgot an "of" before "it" here :
>"I wouldn't even get the reference it if"

into : "I wouldn't even get the reference of it if"

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -625,6 +657,147 @@ mission "Saryd Census 2"
+		event "saryd couple has money" 42 86
+		conversation
+			`Aunaris thanks you for bringing her here, and apologizes for not being able to pay you right away, but promises once she and Inturi have figured out their home, they will work hard to pay you.`
+			`	"Stay here for a few months, I probably will, before the remaining funds I have," she says, before saying goodbye and taking a cable car to her new workplace.`
```
that'd get her sentence to `"Stay here for a few months, I probably will, before the remaining funds, I will have"` which sounds somewhat off to me. added an `acquired` at the end instead, does that work too?

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	destination "Saros"
+	on offer
+		conversation
+			`The couple leaves your ship, and head to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties here, in the most valued (and most sought-after) locations, and that they should still have enough money to pay you afterwards.`
```
```suggestion
			`The couple leaves your ship and heads to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties in the most valued (and most sought-after) locations here, and that they should still have enough money to pay you afterwards.`
```
Double predicate, no comma. Rearranged for better flow.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`The couple leaves your ship, and head to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties here, in the most valued (and most sought-after) locations, and that they should still have enough money to pay you afterwards.`
+			`	They return from consulting, the both of them with disappointed looks on their faces. Inturi starts looking at the ships landing and departing, while Aunaris comes talk to you.`
```
```suggestion
			`	They return from consulting, the both of them with disappointed looks on their faces. Inturi starts looking at the ships landing and departing while Aunaris comes talk to you.`
```
Independent clause followed by dependent clause; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`The couple leaves your ship, and head to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties here, in the most valued (and most sought-after) locations, and that they should still have enough money to pay you afterwards.`
+			`	They return from consulting, the both of them with disappointed looks on their faces. Inturi starts looking at the ships landing and departing, while Aunaris comes talk to you.`
+			`	"Kept watch of one property, for months we had. On a mountain village, overlooking a lake, and with access to a nearby forest, it is. A few days ago, purchased it was," she sighs. "Available, another plot of land neighboring it is, but over half our funds, the land alone costs."`
+			`	Inturi then looks over, as if he had just received a shock, and says, "That's it! A ship! Live in a ship, we could!"`
```
Creative. I like it.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	Inturi then looks over, as if he had just received a shock, and says, "That's it! A ship! Live in a ship, we could!"`
+			choice
+				`	"What do you mean?"`
+				`	"I like the enthusiasm, but unfortunately I'm not looking for any more crewmembers right now."`
```
```suggestion
				`	"I like the enthusiasm, but, unfortunately, I'm not looking for any more crew members right now."`
```

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			choice
+				`	"What do you mean?"`
+				`	"I like the enthusiasm, but unfortunately I'm not looking for any more crewmembers right now."`
+				`	"If you say so. Just don't forget the oranges, scurvy probably is a problem for Saryds too."`
```
Nice little joke/reference.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+				`	"I like the enthusiasm, but unfortunately I'm not looking for any more crewmembers right now."`
+				`	"If you say so. Just don't forget the oranges, scurvy probably is a problem for Saryds too."`
+			`	"Live in a ship?" Aunaris repeats the sentence, puzzled.`
+			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did, often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy, save time and money otherwise spent building a home, it would."`
```
```suggestion
			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did. Often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy. Save time and money otherwise spent building a home, it would."`
```
Both your sentences would be run-on if rearranged back into normal structure.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"Live in a ship?" Aunaris repeats the sentence, puzzled.`
+			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did, often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy, save time and money otherwise spent building a home, it would."`
+			`	They talk for a bit in private, and after a few minutes Inturi apparently managed to fully explain his idea and convince her.`
+			`	They head back to the real estate agency, and arrange the purchase of the plot of land. Inturi then asks that you transport him to <destination>, where the basic Arach ship model is sold, and after buying it he will return here with the new "house."`
```
```suggestion
			`	They head back to the real estate agency and arrange the purchase of the plot of land. Inturi then asks that you transport him to <destination>, where the basic Arach ship model is sold, and after buying it he will return here with the new "house."`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did, often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy, save time and money otherwise spent building a home, it would."`
+			`	They talk for a bit in private, and after a few minutes Inturi apparently managed to fully explain his idea and convince her.`
+			`	They head back to the real estate agency, and arrange the purchase of the plot of land. Inturi then asks that you transport him to <destination>, where the basic Arach ship model is sold, and after buying it he will return here with the new "house."`
+			`	Aunaris will remain here to finish sorting out all the paperwork, and will be waiting by the property once you two return.`
```
```suggestion
			`	Aunaris will remain here to finish sorting out all the paperwork and will be waiting by the property once you two return.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`Inturi heads right to the shipyard, and you help him with the purchase itself. He says that alhough he has worked in many ships over the years, he never went to the shipyard for more than picking up some parts for repair, or to polish a newly produced model.`
+			`	Only when you're with him inside the Arach vessel is that you think of asking if he even knows how to fly a ship, but he reassures you that that won't be a problem.`
```
```suggestion
			`	Only when you're with him inside the Arach vessel do you think of asking if he even knows how to fly a ship, but he reassures you that it won't be a problem.`
```
Replaced a few words to make things cleaner.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`Inturi heads right to the shipyard, and you help him with the purchase itself. He says that alhough he has worked in many ships over the years, he never went to the shipyard for more than picking up some parts for repair, or to polish a newly produced model.`
+			`	Only when you're with him inside the Arach vessel is that you think of asking if he even knows how to fly a ship, but he reassures you that that won't be a problem.`
+			`	"To work as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`
```
```suggestion
			`	"Working as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`
```
Personal preference.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`Inturi heads right to the shipyard, and you help him with the purchase itself. He says that alhough he has worked in many ships over the years, he never went to the shipyard for more than picking up some parts for repair, or to polish a newly produced model.`
+			`	Only when you're with him inside the Arach vessel is that you think of asking if he even knows how to fly a ship, but he reassures you that that won't be a problem.`
+			`	"To work as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`
+			`	He says he will wait for your lead, and follow you to <destination> once you depart.`
```
```suggestion
			`	He says he will wait for your lead and follow you to <destination> once you depart.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		event "saryd couple follow-up" 51 96
+		conversation
+			`You help Inturi park the ship on the plot of land, but since the Arach Courier takes up almost the entire area, your own ship has to land on a regular landing pad.`
+			`	After taking a transport up the mountain, you see that some workers are already there, and are starting to remove ship components that won't be needed, like its engines and the hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`
```
```suggestion
			`	After taking a transport up the mountain, you see that some workers are already there and are starting to remove ship components that won't be needed, like its engines and hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`
```
Double predicate; no comma. Removed unnecessary `the`.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`You help Inturi park the ship on the plot of land, but since the Arach Courier takes up almost the entire area, your own ship has to land on a regular landing pad.`
+			`	After taking a transport up the mountain, you see that some workers are already there, and are starting to remove ship components that won't be needed, like its engines and the hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`
+			`	The couple comes meet up with you, "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`
```
```suggestion
			`	The couple comes to meet up with you. "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`
```

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	The couple comes meet up with you, "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`
+			`	"Helped us a lot in getting it all sorted out you have, Captain <last>. Focus on working hard to pay you back in full, we will," Inturi adds.`
+			choice
+				`	"You don't need to worry about paying me, I helped because I wanted to, not for a payday."`
```
```suggestion
				`	"You don't need to worry about paying me. I helped because I wanted to, not for a payday."`
```
You might be able to use Amazinite's new payment in conversation feature here. Seems like you have it all worked out here, but just thought I would let you know.

Arachi-Lover replied:
when that is merged

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+					goto end
+			apply
+				set "refused saryd couple payment"
+			`	They look at you confused and then try to insist on paying you, but after some minutes they agree. "Too kind a heart you are, Captain," Aunaris says.`
```
```suggestion
			`	They look at you confused and then try to insist on paying you, but after some minutes they agree. "Too kind a heart you are, captain," Aunaris says.`
```

Arachi-Lover replied:
same case as before

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`When you land, you're notified of <payment> being transferred to your account.`
+			`	With it, there's a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`
```
```suggestion
			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
```
No comma b/c dependent clause following independent clause. `The couple's home` is a personal preference.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`When you land, you're notified of <payment> being transferred to your account.`
+			`	With it, there's a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`
+			`	They thank you again, and say you're welcome to visit anytime.`
```
```suggestion
			`	They thank you again and say you're welcome to visit anytime.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		government "Coalition" "Heliarch"
+	on offer
+		conversation
+			`When you land, see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
```
```suggestion
			`When you land, you see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
```

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`When you land, see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become some ...
```
```suggestion
			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
```
Same edits as above.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`When you land, see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become some ...
+			`	They thank you again, and say you're welcome to visit anytime.`
```
```suggestion
			`	They thank you again and say you're welcome to visit anytime.`
```

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`You see some Arachi talking among themselves in the spaceport, occasionally glancing at and pointing at you.`
+			`	One of them finally comes to you, and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`
```
```suggestion
			`	One of them finally comes to you and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+				`	"You've found the perfect person for the job. I'm in."`
+				`	"I suppose that's true, but I'm not looking for courier jobs, sorry."`
+					decline
+			`	The other Arachi come over after she signals to them that you've accepted to help, and they come thank you. They say that after you've picked up the vaccines on <planet> you'll have to bring them to some planets, and wish you a good trip.`
```
```suggestion
			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the vaccines on <planet>, you'll have to drop them off on another planet before wishing you a good trip.`
```
General edits to fix a couple grammar issues and improve flow.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+
+mission "Longcow Vaccines 2"
+	name "Longcow Vaccines"
+	description "Take the <cargo> to <destination>, so that the Arachi ranchers can vaccinate their cattle of longcows."
```
```suggestion
	description "Take the <cargo> to <destination> so that the Arachi ranchers can vaccinate their cattle of longcows."
```
No comma b/c dependent clause follows independent.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	destination "Corral of Meblumem"
+	on offer
+		conversation
+			`The dockworkers load <cargo> into your cargo hold, after verifying you've been hired by the Arachi cooperative.`
```
```suggestion
			`The dockworkers load <cargo> into your cargo hold after verifying you've been hired by the Arachi cooperative.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`The dockworkers load <cargo> into your cargo hold, after verifying you've been hired by the Arachi cooperative.`
+			`	A bit after the last crate has been put there, you're contacted by the Arachi, and they say you should bring the vaccines to <destination>, where they will pay you <payment>.`
```
```suggestion
			`	A bit after the last crate has been placed, you're contacted by the Arachi, and they say you should bring the vaccines to <destination>, where they will pay you <payment>.`
```

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`You're paid 412,000 credits when you land, but as you help the ranchers unload the vaccines, they say that something is off.`
+			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`
```
```suggestion
			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours, is not."`
```
Added comma for reverse-sentence structure consistency.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`You're paid 412,000 credits when you land, but as you help the ranchers unload the vaccines, they say that something is off.`
+			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`
+			`	She goes away for about ten minutes, to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`
```
```suggestion
			`	She goes away for about ten minutes to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`
```
No comma b/c dependent clause follows independent.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`
+			`	She goes away for about ten minutes, to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`
+			`	"Mixed up the cargo, they have. His own longcow ranch on <destination>, a cousin of mine has," she says. "Sent his batch of vaccines along with my own, they have."`
+			`	They contact the cousin in question, and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`
```
```suggestion
			`	They contact the cousin in question and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	destination "Chosen Nexus"
+	on offer
+		conversation
+			`An elderly Arach beckons you as you enter the spaceport. He is accompanied by four Arachi children, as they all stand close to a large pile of luggage.`
```
```suggestion
			`An elderly Arach beckons to you as you enter the spaceport. He is accompanied by four Arachi children, who all stand close to a large pile of luggage.`
```

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`An elderly Arach beckons you as you enter the spaceport. He is accompanied by four Arachi children, as they all stand close to a large pile of luggage.`
+			`	He explains that he used to run an orphanage here, but now has gotten too old for the job, and has arranged for the four kids to be transferred to a foster home on <destination>.`
```
```suggestion
			`	He explains that he used to run an orphanage here, but has now gotten too old for the job. He has arranged for the four kids to be transferred to a foster home on <destination>.`
```
Sentence was long and maybe a run-on.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`An elderly Arach beckons you as you enter the spaceport. He is accompanied by four Arachi children, as they all stand close to a large pile of luggage.`
+			`	He explains that he used to run an orphanage here, but now has gotten too old for the job, and has arranged for the four kids to be transferred to a foster home on <destination>.`
+			`	"Take care of them, the Saryds there will," he says, patting the head of the youngest and shortest of the kids. "Retiring, I am, but abandon them, I will not. An apartment close to the foster home, I will purchase."`
+			`	He offers to pay you <payment> to bring the five of them, and their <cargo> to <destination>.`
```
```suggestion
			`	He offers to pay you <payment> to bring the five of them and their <cargo> to <destination>.`
```
Only two items in this list; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			choice
+				`	"Of course I'll help. Here, let me help you get the luggage into my ship."`
+					accept
+				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`
```
```suggestion
				`	"I hope things work out for you, but I'm actually not going be around Coalition space for long now."`
```
Just seems smoother to me. Your choice.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		"coalition jobs" ++
+		payment 268000
+		conversation
+			`You help the old Arach caretaker and some workers take the luggage out of your ship, then head to the orphanage.`
```
```suggestion
			`You help the old Arach caretaker and some workers take the luggage out of your ship then head to the orphanage.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		payment 268000
+		conversation
+			`You help the old Arach caretaker and some workers take the luggage out of your ship, then head to the orphanage.`
+			`	A Saryd couple runs the place, and greet you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child, and some Kimek too.`
```
```suggestion
			`	A Saryd couple runs the place and greets you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child and some Kimek too.`
```
Double predicate; no comma. Also, changed `greet` to singular to match `runs`.
List only has two items; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`You help the old Arach caretaker and some workers take the luggage out of your ship, then head to the orphanage.`
+			`	A Saryd couple runs the place, and greet you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child, and some Kimek too.`
+			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside, to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
```
```suggestion
			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
```
Dependent clause follows independent clause; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+mission "Arachi Cartoonist 1"
+	minor
+	name "A New Design"
+	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs for his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
```
```suggestion
	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
```
Changed to avoid `for` used twice close together.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you on your tracks.`
+			`	He introduces himself as Drugub, a cartoonist who has seen some local fame lately for his sci-fi comics. "Many ship designs produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
```
```suggestion
			`	He introduces himself as Drugub, a cartoonist of local fame for his sci-fi comics. "Many ship designs produced in our space are fascinating, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
```

Arachi-Lover replied:
his saying "Many ship designs produced in our space are" translates to "Many ship designs are produced in our space", that is, to state that there's what he judges to be a large variety of ships produced there

Wizrad36 replied:
Oh, that makes more sense. You might want to consider rewording that sentence considering I thought it was an error when reading and not just their way of speaking. Could just be me, though.

Arachi-Lover replied:
changed some words to try and clarify

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+				`	"Sorry, but I won't be heading back to human space for a while."`
+					decline
+			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
+			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs, and settles for those.`
```
```suggestion
			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs and settles for those.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	destination "Mebla's Portion"
+	on offer
+		conversation
+			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer, which you then unfold to see all of Betelgeuse's ships depicted, as well as some details on each (and the pricing for each too, of course).`
```
```suggestion
			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer. Inside, all of Betelgeuse's ships are depicted, and some details on each (along with the pricing, of course) are included.`
```
Long sentence. Edits to improve flow.

Arachi-Lover replied:
didn't follow it exactly. if that caused a new problem do mention it

Wizrad36 replied:
Good, except remove the comma after `depicted`. Independent clause followed by a dependent clause.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer, which you then unfold to see all of Betelgeuse's ships depicted, as well as some details on each (and the pricing for each too, of course).`
+			`	"Usually we show these to first time buyers, but you can take one if you want. Are you here for a friend like that? Maybe I could help you figure out what ship would best suit them."`
+			`	You thank her, saying that your friend does not have the funds for any of the ships, and just wants some info, and head back to your ship.`
```
```suggestion
			`	You thank her, saying that your friend does not have the funds for any of the ships and just wants some info, then head back to your ship.`
```
Double predicate; no comma.
Thought `then` fits better than `and`, but your choice.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		"coalition jobs" ++
+		payment 75940
+		conversation
+			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`
```
```suggestion
			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's a scroll full of ancient knowledge.`
```
Personal preference.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		payment 75940
+		conversation
+			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`
+			`	He hands you your payment of <payment>, and spends some time with you still, asking you all the questions he can think of about each ship.`
```
```suggestion
			`	He hands you your payment of <payment> and spends some time with you still, asking you all the questions he can think of about each ship.`
```
Double predicate; no comma.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`
+			choice
+				`	"Of course! It's called <ship>."`
+				`	"The name is nothing special, I wouldn't even get the reference of it if I read your comic in the future honestly."`
```
```suggestion
				`	"The name is nothing special. I wouldn't even get the reference of it if I read your comic in the future, honestly."`
```
Run-on sentence. Not sure if you _need_ a comma for `honestly` or not, but it sounds better to me personally. Up to you.

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+				`	"Of course! It's called <ship>."`
+				`	"The name is nothing special, I wouldn't even get the reference of it if I read your comic in the future honestly."`
+					goto noship
+			`	"<ship> it is! Do it justice I will, Captain. Thanks again."`
```
```suggestion
			`	"<ship> it is! Do it justice, I will, captain. Thanks again."`
```
Added a comma to follow reverse-sentence structure rules.

Arachi-Lover replied:
same as before on Captain

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"<ship> it is! Do it justice I will, Captain. Thanks again."`
+				accept
+			label noship
+			`	"Ah, I see. Well, just a reference it would be, so as you say not very special it would be I suppose. Anyway, thanks again Captain."`
```
```suggestion
			`	"Ah, I see. Well, just a reference it would be, so as you say, not very special it would be, I suppose. Anyway, thanks again, captain."`
```
Don't really like that middle sentence. You might consider rewriting it in a more concise way.

Arachi-Lover replied:
changed the middle a bit; same thing on Captain as previously mentioned

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		conversation
+			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`
```
some planets, specifically a few in Coalition space itself, touch on how the day-night cycle works on them, I just added this bit here to be something like that, only in a mission rather than a planet description.

Wizrad36 replied:
Fair enough.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`
+			`	"Accepted I was! Grant me the rank, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`
```
I'm more inclined to keep that first sentence without a , since there's the usage of a !, so it would be something said in a quick and excited fashion

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		event "coalition virus contained" 72
+		conversation
+			`As you enter the spaceport, you notice several battalions of Coalition doctors roaming about, talking with merchant captains and usually then following them to their ships.`
+			`	One such crowd approaches you, and one of them says, "Broken out on <destination>, a deadly virus has. Captains willing to transport us and <cargo>, we seek." She shows you to the group of doctors, <bunks> of them.`
```
didn't use that one in specific but should be fixed now

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	on visit
+		dialog `You've reached <planet>, but the doctors and the medical supplies haven't arrived yet. Depart and wait for the ships carrying them.`
+	on complete
+		payment 768540
```
I still haven't figured out how to make non-specific payments work so I just use set values

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		government "Coalition"
+		ship "Saryd Sojourner" "Noset Temu"
+		ship "Saryd Sojourner" "Miest Laba"
+		ship "Arach Hulk" "Bloobrupblot"
```
those are Coalition ship names, I just departed and waited for 2 Saryd and an Arach ship and wrote those down

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`You and the other captains are told to wait in one of several lines formed here. As they talk with others and then with you, they mention how many of the captains here just came back from delivering vaccines to Arachi and Saryd space, to the travelers who got infected before going to planets there.`
+			`	The line moves quickly, and once it's your turn, you're paid <payment>.`
+			`	Looking back at the line, and seeing the sheer number of captains waiting for their share, it's no wonder the Kimek that paid you were extremely brief, but as you come back to your ship, you see that a message was sent to you by the Coalition government. More specifically, it appears to have been sent by their Department of Health.`
+			`	"To all captains who helped in dealing with the recent viral outbreak, you have our gratitude, and may rest tonight knowing you've contributed to the continued prosperity of our Coalition."`
```
I'm not sure on the `well`

Wizrad36 replied:
Up to you, shouldn't change the sentence in any drastic way.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		"coalition jobs" ++
+		conversation
+			`Apparently this Saryd in question doesn't live here on the spaceport village, so you must take a train to go to his home. Thankfully a train ticket was attached to the package, so you waste no time and get on the next train there.`
+			`	Even for a world without too large a population, the train wagons here feel unusually empty, with only a handful of other passengers on the one you settle in. The trip takes a few hours, so you have plenty of time to look out the window and gaze at the sights of the seemingly infinite winter forest.`
```
is the change necessary?

Wizrad36 replied:
Nope, just tried to simplify what was being said. Perfectly fine to keep as is.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	They help you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
+			`	"No luck?" the translation machine used by the one who bumped into you translates, still active.`
+			`	His colleagues shake their heads, "Shipping here, the Arachi no longer are."`
+			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant Captain you are, correct? Looking for work, you are?"`
```
can someone confirm the change? since Captain is always capitalized when used to refer to the player, I did so here.

Wizrad36 replied:
After looking into it, I don't think this one is capitalized. A good rule of thumb might be to ask if `<first> <last>` could be substituted in place of `captain`. That's just something I just came up with, so it might not be correct 100% of the time. If so, it's capitalized, and if not, it's lowercase.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+				`	"Actually I'm just here for the view, sorry."`
+					decline
+			`	They all start laughing, some clearly finding it much funnier than you anticipated. They happen to be the same ones from whom you notice the stronger scent of alcohol.`
+			`	"Fair enough. Not a common sight here, humans are, Captain," the one with a translation box says. "Asking for such a response, I was."`
```
this one for sure is capitalized, you're being directly addressed, and when Captain is used in such a case it's capitalized in the game

Wizrad36 replied:
Yeah, just did some google searching and you're right on this one. Apparently when it's substituted in place of a person's in this way it's a proper noun, and thus capitalized. My bad.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			label work
+			`	They tell you that they are students at Starlit University, and will be graduating soon. They intend on throwing a party to celebrate, and had the idea of buying some traditional Arachi Rum for that.`
+			`	"Stopped shipping it here, the Arachi have," another Saryd says, pulling on the translation box on his friend's neck, nearly yanking it off. "Happened with many other drinks, this also has."`
+			`	They explain that the rum is produced on <destination>, and that you'll probably have an easier time finding it there. Though you can't know for sure how many people they intend on inviting to this party, and although you don't know how much a Saryd can drink, you still worry a bit when they say that "two tons should do."`
```
it read quite weird so I kinda rewrote it, any new issues now?

Wizrad36 replied:
Looks good to me!

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"Remained stable, the Saryd population has," one of them comments. "At about 30 billion individuals, our numbers still are, though found a very slight increase in fertility rate has been."`
+			`	As you continue to chat, one of them gets curious and asks you about humanity's population, to which you respond that your species numbers in about 100 billion people.`
+			`	"Double that of the Arachi," one of the workers says.`
+			`	"And half that of the Kimek," another one completes.`
```
you can see the exact numbers of each species here https://github.com/endless-sky/endless-sky/issues/2540 , given by MZ himself. you gotta remember that the Coalition also has a complete Quarg ringworld, and 2 incomplete ones. the complete one can hold at least 1 Trillion individuals, so they're better off in total living space than humanity I'd wager

Wizrad36 replied:
The scale of some of the stuff always throws me off. Sounds like you're right.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+		"coalition jobs" ++
+		event "saryd couple has money" 42 86
+		conversation
+			`Aunaris thanks you for bringing her here, and apologizes for not being able to pay you right away, but promises once she and Inturi have figured out their home, they will work hard to pay you.`
```
added a sentence at the end mentioning the pay will be later. any issues in that new portion?

Wizrad36 replied:
Nope, all good.

Arachi-Lover commented:
Some of the resolved conversations were discussed in PMs, just to clarify on the stuff that's not marked as such and I haven't commented on, I'm not very inclined to go with the suggestions rn so I might rewrite those bits and ask if the new sentences are alright. some of the stuff though, the "commas for consistency", felt like they were just making the dialogue robotic, so for those I don't think I'll change

Wizrad36 commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+				`	"Oh no, I live here actually. In the humantown district, right around the corner."`
+				`	"Actually I'm just here for the view, sorry."`
+					decline
+			`	They all start laughing, some clearly finding it much funnier than you anticipated. They happen to be the same ones from whom you notice the stronger scent of alcohol.`
```
All good if you don't want to use my substitution here, but I would still recommend some change to the line for the reason stated above.

Arachi-Lover replied:
tried redoing it but not sure if I was very successful.

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
+	to offer
+		has "Coalition: First Contact: done"
+		random < 95
```
You can remove this.

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	destination "Prime"
+	on offer
+		conversation
+			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you on your tracks.`
```
```suggestion
			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you in your tracks.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	source "Mebla's Portion"
+	destination "Prime"
+	on offer
+		conversation
```
I feel like this conversation could stand to be improved. Right now as it is, there seems to be little introduction of this character. Perhaps he shows you some of his comics to get you interesting. Then when you say you're interested in helping, it seems to me like you would be bringing him to human space, but instead we get a "Okay, that sounds good. Good luck traveling across the galaxy for me!" Maybe instead bring him as a passenger to see the ships first hand. If he can't leave Coalition space, perhaps mention something about not being allowed to leave by the Heliarchs, only asking that you take pictures or get a hologram to show to him.

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	destination "Mebla's Portion"
+	on offer
+		conversation
+			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer. Inside, all of Betelgeuse's ships are depicted with some details on each (along with the pricing, of course).`
```
```suggestion
			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships," she hands you a flyer. Inside, all of Betelgeuse's ships are depicted with some details on each (along with the pricing, of course).`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer. Inside, all of Betelgeuse's ships are depicted with some details on each (along with the pricing, of course).`
+			`	"Usually we show these to first time buyers, but you can take one if you want. Are you here for a friend like that? Maybe I could help you figure out what ship would best suit them."`
```
```suggestion
			`	"Usually we show these to first time buyers, but you can take one if you want. Is your friend interested in buying? Maybe I could help you figure out what ship would best suit them."`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+			`	He hands you your payment of <payment> and spends some time with you still, asking you all the questions he can think of about each ship.`
+			`	"Very beautiful, this 'Leviathan' is. Thinking of many ways to draw it, I already am," he says. "Maybe a flagship of one of my characters, make it I will..."`
+			`	He spends some more minutes listening to you after he asks that you describe how you've seen each of the ships perform, thanks you again, and prepares to leave.`
+			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`
```
```suggestion
			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan,' may I? Just as a small thank you."`
```

Amazinite commented:
Maybe have these mission increment the coalition jobs condition by more than just one. That way doing spaceport missions is quicker than doing jobs.

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	minor
+	name "Acquire Arachi Rum"
+	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi Rum, currently only sold on <destination>."
+	deadline
```
added

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+			`	He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
+			`	"No luck?" the translation machine used by the one who bumped into you translates, still active.`
+			`	His colleagues shake their heads, "Shipping here, the Arachi no longer are."`
+			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant captain you are, correct? Looking for work, are you?"`
```
did thing to change it up, added a branch so that you do get their words in case the player has the Coalition language

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+		conversation
+			`You and the other captains are told to wait in one of several lines formed here. As they talk with others and then with you, they mention how many of the captains here just came back from delivering vaccines to Arachi and Saryd space, to the tourists who contracted the disease before flying to those areas.`
+			`	The line moves quickly, and once it's your turn, you're paid <payment>.`
+			`	Looking back at the line, and seeing the sheer number of captains waiting for their share, it's no wonder the Kimek that paid you were extremely brief. As you come back to your ship, you see that a message was sent to you by the Coalition government. More specifically, it appears to have been sent by their Department of Health.`
```
added thing

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+		payment 2593230
+		"coalition jobs" ++
+		conversation
+			`You and the other captains are told to wait in one of several lines formed here. As they talk with others and then with you, they mention how many of the captains here just came back from delivering vaccines to Arachi and Saryd space, to the tourists who contracted the disease before flying to those areas.`
```
redid to try and make it readable

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`At any given time there are thousands of Kimek here in the spaceport, either mingling among themselves during a break from work or negotiating travel prices with starship captains. You imagine that if you weren't a human and clearly stood out, you too would be constantly getting offers to travel to another world, as many civilian captains here wish to fill a last bunk or two they have vacant  ...
+			`	You manage to find a restaurant here where you spot some food you've already learned you can eat, so you take a seat and order something.`
```
dunno, this may well be the last mission a player ever comes across in Coalition space, and by that point if they have been only in Coalition space doing missions and taking jobs for some ingame months it'd be odd to have this one say "you finally decide to try the food here"

Arachi-Lover commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
+			`	When a Heliarch agent passes by the entrance, the Kimek says something in his language, heading to meet the Heliarch, and speaking with them briefly. The Heliarch agent then accompanies the Kimek waiter to your table.`
+			choice
+				`	"Is there an officer problem?"`
+				`	"Did I do something wrong?"`
```
just removed the joke option and replaced with another neutral one

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+					decline
+			label accept
+			`	He thanks you, and this time follows you right away, as he already has his bags prepared.`
+			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he isn't aiming for the more military-focused side of the government.`
```
Instead of saying what he *doesn't* want to do, say what he *does*, and contrast that with what he doesn't.
```suggestion
			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work in the scientific branch of the government, instead of the more military-focused side.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+	destination "Weir of Glubatub"
+	on offer
+		conversation
+			`While walking by some shops, young Saryd bumps into you as you're turning a corner, knocking you to the ground.`
+			`	He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
```
```suggestion
			`While walking by some shops, a young Saryd bumps into you as you're turning a corner, knocking you to the ground. He helps you up, apologizing. A couple others that were with him entered a shop without even noticing you, but almost immediately come back out.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+			`While walking around the spaceport, you notice one of the Saryd workers that was refueling your ship is quickly galloping somewhere, only without his uniform, as if his shift just ended. He meets up with another Saryd and starts looking at a map with her.`
+			choice
+				`	(Approach them.)`
+				`	(Go back to your ship and check if it was properly refueled.)`
+					decline
```
I think that this choice should lead to some writing where you find that your ship is fine and you decide to move on with your day, instead of just outright ending here.

Arachi-Lover replied:
added little bit at the end

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <planet>, you'll have to drop them off on another planet before wishing you a good trip.`
+				accept
+	on stopover
+		dialog `The dockworkers load <cargo> into your cargo hold after verifying you've been hired by the Arachi cooperative. A bit after the last crate has been placed, you're contacted by the Arachi, and they say you should bring the vaccines to <destination>, where they will pay you <payment>.`
```
```suggestion
		dialog `The dockworkers load <cargo> into your cargo hold after verifying you've been hired by the Arachi cooperative. A bit after the last crate has been placed, you're contacted by the Arachi, and they say you should bring the antibiotics to <destination>, where they will pay you <payment>.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`You're paid 421,000 credits when you land, but as you help the ranchers unload the vaccines, they say that something is off.`
+			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`
```
```suggestion
			`	"Only 6 tons of antibiotics, we requested," one of them says. "That extra ton surplus, ours is not."`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+	destination "Nearby Jade"
+	on offer
+		conversation
+			`You're paid 421,000 credits when you land, but as you help the ranchers unload the vaccines, they say that something is off.`
```
```suggestion
			`You're paid 421,000 credits when you land, but as you help the ranchers unload the antibiotics, they say that something is off.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`
+			`	She goes away for about ten minutes to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`
+			`	"Mixed up the cargo, they have. His own longcow ranch on <destination>, a cousin of mine has," she says. "Sent along with my own, his batch of antibiotics was."`
+			`	They contact the cousin in question and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`
```
```suggestion
			`	They contact the cousin in question and he asks that you transport his own part of the antibiotics to <destination>, where he will pay you <payment>.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+	on complete
+		"coalition jobs" += 2
+		payment 193000
+		dialog `You land and meet up with the cousin of the Corral of Meblumem rancher, and he pays you <payment>. "Mix up our batches due to our family name, the doctors always do!" he grumbles as the vaccines are being removed from your cargo hold.`
```
```suggestion
		dialog `You land and meet up with the cousin of the Corral of Meblumem rancher, and he pays you <payment>. "Mix up our batches due to our family name, the doctors always do!" he grumbles as the antibiotics are being removed from your cargo hold.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+				`	"You've found the perfect person for the job. I'm in."`
+				`	"I suppose that's true, but I'm not looking for courier jobs, sorry."`
+					decline
+			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <planet>, you'll have to drop them off on another planet before wishing you a good trip.`
```
So you've changed this mission to use a stopover, but this wording here is still very roundabout. 
```suggestion
			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <stopovers>, you'll have to drop them off on <planet> before wishing you a good trip.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+		conversation
+			`You see some Arachi talking among themselves in the spaceport, occasionally glancing at and pointing at you.`
+			`	One of them finally comes to you and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`
+			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest antibiotics for the cattle on <destination>.`
```
```suggestion
			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest antibiotics for the cattle on <stopovers>.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+	destination "Prime"
+	on offer
+		conversation
+			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you in your tracks.`
+			`	He introduces himself as Drugub, a cartoonist of local fame for his sci-fi comics. "Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
+			choice
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
+					decline
+			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
+			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs and settles for those.`
+			`	He thanks you and wishes you safe travels on your journey to <planet>.`
+				accept
```
Got bored. Wrote this.
```suggestion
			`As you're moving along one of the underground tunnels that connect the spaceport here, you notice an Arach off in the distance looking at you. He shouts something in Arachi, then begins running toward you while fiddling with a translation box.`
			`	By the time he reaches you he is moving much slower than when he started. "Vistitor, forgive me," he says, breathing heavily while resting against the nearest wall. "Miss this opportunity to meet you, I could not."`
			`	The Arach takes a deep breath and stands upright. "Drugub, I am. A creator of science fiction comics, I am famous for. Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would. Travel to human space, you may, and return with images for inspirations?"`
			choice
				`	"I can help with that. Do you have any type of ship in mind?"`
				`	"Sorry, but I won't be heading back to human space for a while."`
					decline
			`	Drugub lets out a noise that you would assume is a laugh. "Of the ships of your people, I know not. Please describe them."`
			`	You describe the various shipyards of human space; Lionheart, who develop some of the most advanced ships in human space; Tarazed, who design their ships with inspiration from the Quarg (resulting in a noticeable grimace from Drugub); Syndicate, who specialize in cheap yet (somewhat) effective ships.`
			`	After describing the ships of Betelgeuse Shipyards on <planet> and how some of their designs are almost as old as humanity's hyperdrive, he stops you. "Perfect, this would be. Set during a... less developed time, my series is. Older designs, I have been searching for."`
			choice
				`	"Okay. I'll look for something to bring back for you."`
					goto end
				`	"Sounds like a plan. Let's get ready to leave."`
			`	"Us?" Drugub asks, letting out another noise that you think is a laugh. "No, only you may go. Leave Coalition space, the Heliarchs do not allow. Permission, I require. Without me, you must go. Bring images for me is all I need."`
			label end
			`	He thanks you and wishes you safe travels on your journey to <destination>.`
				accept
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -509,8 +509,7 @@ mission "Saryd Students 1"
 		conversation
-			`While walking by some shops, young Saryd bumps into you as you're turning a corner, knocking you to the ground.`
-			`	He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
+			`While walking by some shops, young Saryd bumps into you as you're turning a corner, knocking you to the ground. He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
```
```suggestion
			`While walking by some shops, a young Saryd bumps into you as you're turning a corner, knocking you to the ground. He helps you up, apologizing. A couple others that were with him entered a shop without even noticing you, but almost immediately come back out.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -250,7 +250,7 @@ mission "Kimek Student 2"
 			label accept
 			`	He thanks you, and this time follows you right away, as he already has his bags prepared.`
-			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he isn't aiming for the more military-focused side of the government.`
+			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work for for the scientific branch of the government, instead of the more military-focused side.`
```
```suggestion
			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work for the scientific branch of the government, instead of the more military-focused side.`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,856 @@ mission "Coalition License Patch"
+	on offer
+		conversation
+			`As you're moving along one of the larger underground tunnels that connect the spaceport here, a dozen makeshift stands, each with a different apparent 'theme' to them, captures your attention. When you get closer, it's clear that the event is something of a convention, each stand managed by a different artist that lives here.`
+			`	After you're done looking at a stand that showcased various fantasy attires and move on to the neighboring one, it's owner jumps as soon as he turns to see you, looking dumbfounded. He introduces himself as Drugub, a cartoonist of local fame for his science fiction comics, showing various examples of them as he speaks with you. "Many styles of ships produced in our space are, but only so man ...
+			choice
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
```
I'd change this second choice to not mention going to human space, as it isn't immediately apparent to me that that is what Drugub is asking. (It could be that he just wants to see my ship.)

Arachi-Lover replied:
changed it up

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,856 @@ mission "Coalition License Patch"
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
+					decline
+			`	He looks puzzled, "For a variety of purposes, ships in my comics are used, so to know of a variety of ships, preferrable it would be."`
```
```suggestion
			`	He looks puzzled. "For a variety of purposes, ships in my comics are used, so to know of a variety of ships, preferable it would be."`
```

Amazinite commented on data/coalition/coalition missions.txt:
```diff
@@ -143,3 +143,856 @@ mission "Coalition License Patch"
+			choice
+				`	"In that case, I'll head to their home shipyard and see what I can bring back for you."`
+					goto end
+				`	"Alright, it's a bit of a long trip to <planet> but if you're sure, let's head to my ship and I'll show you your bunk."`
```
```suggestion
				`	"Alright, it's a bit of a long trip to <planet>, but if you're sure, let's head to my ship and I'll show you your bunk."`
```
